app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -386,10 +386,17 @@ def remove_middleware_datasource(ems_ref)
     end
 
     def emit_middleware_notification(type, op_name, op_arg, ems_ref)
+      mw_deployment = MiddlewareDeployment.find_by(:ems_ref => ems_ref)
+      mw_server = if mw_deployment.nil?
```

Comment:
:) Unfortunately most times my local rubocop shows different errors than miq-bot so I end up not running it. Should be good now!